analysis/crossref/crossref.go

```diff
@@ -70,3 +77,35 @@ func writeXRef(res *database.Resources, collection string,
 	}
 	externWG.Done()
 }
+
+func finalizeXRef(res *database.Resources, collection string) {
```

Comment:
Modules may contain duplicate entries for certain ip addresses. For example two sources could beacon to a single destination server. The destination server would be recorded twice in the beaconing results. Here we group the xref results by ip address and build a unique list of the modules the ip address was found in.